src/modules/utils/Utils.ts

```diff
@@ -110,4 +111,20 @@ export class Utils
 
         return res;
     }
+
+    /**
+     *  Gets the path to where the execution command was entered for this process.
+     *  If this value is not set, returns the root path of the app.
+     */
+    public static getInitCWD (): string
+    {
+        //  A type of process.env.INIT_CWD is string | undefined.
+        //  In order to use this, it is always necessary to make sure
+        //  that it is `undefined`.
+        //  `string` can be return only when it is not `undefined`.
+        if (process.env.INIT_CWD === undefined)
```

Comment:
Using 'process.env.INIT_CWD' found by @Geod24 , the working directory could be found.